app/controllers/concerns/billable_event_trackable.rb

```diff
@@ -0,0 +1,54 @@
+module BillableEventTrackable
+  extend ActiveSupport::Concern
```

Comment:
we're not using `included` or anything, so I don't we need to extend this, looks like a plain mixin to me

```suggestion
```